lib/services/messaging_services/dms_api_service.dart

```diff
@@ -39,4 +40,62 @@ class DMApiService {
       return null;
     }
   }
+
+  Future<String> roomCreator(String friendId) async {
+    final orgId = _userService.currentOrgId;
+    final String memberID =
+        storageService.getString(StorageKeys.idInOrganization).toString();
+    String endpoint = '/v1/org/$orgId/users/$memberID/room';
+    var body = {
+      "org_id": orgId,
+      "room_member_ids": [memberID, friendId],
+      "room_name": "Prince"
```

Comment:
Prince? As room name

Reply:
It's temporary until we can get the other users display name

Reply:
You can still use the email

Reply:
I used display name